internal/ias/client_test.go

```diff
@@ -471,10 +533,11 @@ func mockDeleteApplicationWithResponseStatusNotFound(clientMock *mocks.ClientWit
 		}, nil)
 }
 
-func mockGetTokenUrl(t *testing.T, tokenUrl *string) *oidcmocks.Client {
+func mockClient(t *testing.T, tokenUrl, jwksURI *string) *oidcmocks.Client {
 	clientMock := oidcmocks.NewClient(t)
-	clientMock.On("GetTokenEndpoint", mock.Anything).
-		Return(tokenUrl, nil)
-
+	clientMock.On("GetTokenEndpoint", mock.Anything).Return(tokenUrl, nil)
+	if tokenUrl != nil {
```

Comment:
why do you want the check `if tokenUrl != nil`?

Reply:
Changed.